emcc.py

```diff
@@ -1724,6 +1724,9 @@ def phase_linker_setup(options, state, newargs, user_settings):
       assert not settings.EXPORTED_FUNCTIONS
       settings.EXPORTED_FUNCTIONS = ['_main']
 
+  if '_main' in settings.EXPORTED_FUNCTIONS:
+    settings.EXPORTED_FUNCTIONS.append('___main_argc_argv')
```

Comment:
Do we export two names in this case, the short and the long?

Reply:
Adding to `EXPORTED_FUNCTIONS` at this point (rather than via the command line) means "export if defined" so only one or the other will be exported.